### PR TITLE
add m4 dependency too google-protobuf

### PR DESCRIPTION
**Pull request by stanley-cheung**
- apparently the sequence matters

### CODE REVIEW

**Comment by stanley-cheung** [comment]
Fixes issue #31

**Comment by tbetbetbe** [comment]
Can we also update the README.md to mention that python-dev is a prerequisite ?

**Comment by stanley-cheung** [comment]
Updated README.md

File: README.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ https://goo.gl/getgrpc
 $ curl -fsSL https://goo.gl/getgrpc | bash -
 ```
 - To install gRPC Python (it installs the gRPC C core if needed)
-  - Prequisite: The installer uses pip to install the grpc [python package][], so either
+  - Prequisite: python-dev, and pip. The installer uses pip to install the grpc
```
Comment (tbetbetbe) [inline README.md]: can you fix the typo :)
s/prequisite/prerequisite/

**Comment by stanley-cheung** [comment]
Fixed typo :)

**Pull request merged**